components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -952,6 +1022,28 @@ def getAnnotation(self, ns=None):
             return AnnotationWrapper._wrap(self._conn, rv[0].child, link=rv[0])
         return None
 
+    def getAnnotationCount(self):
```

Comment:
So I think we do not want to move `self.annotation_counter` but the method could be unified in the main gateway so we don't need to deprecate anything. Happy to chat